samples/Azure.Management.Storage/Generated/Models/PrivateEndpointConnectionListResult.Serialization.cs

```diff
@@ -16,7 +16,7 @@ internal partial class PrivateEndpointConnectionListResult
     {
         internal static PrivateEndpointConnectionListResult DeserializePrivateEndpointConnectionListResult(JsonElement element)
         {
-            Optional<IReadOnlyList<PrivateEndpointConnectionData>> value = default;
```

Comment:
Can we add *ListResult as well?  I know its internal now but can be public later?

Reply:
I've added `PrivateEndpointConnectionListResult` and `PrivateLinkResourceListResult` into the list. I think even if they are not public later, it's good for consistency.